### PR TITLE
Add first report post

### PR DESCRIPTION
**Pull request by BenJam**
Hey @coderberry I think this is ready to roll sans illustration. We will pick up the design as part of a separate site we build around the report and posts like these. Thanks for helping keeping Mr. Hyde under wraps. 

<a href="https://gitpod.io/#https://github.com/sustainers/website/pull/303"></a>

### CODE REVIEW

**Comment by coderberry** [comment]
@BenJam I added the blog section to the website, and updated your post a bit. It's not done 100% yet. I think it could use some styling love (I plan on doing this), but if we want to release it sooner than later, I think we're close.

**Comment by coderberry** [comment]
@jdorfman I can't get the CI to pass on this one. The page looks good and I think we are ok to deploy, but the checks are failing.

**Comment by jdorfman** [comment]
> @jdorfman I can't get the CI to pass on this one. The page looks good and I think we are ok to deploy, but the checks are failing.

Checking. cc @XhmikosR

**Comment by jdorfman** [comment]
@coderberry 

```
"file:/home/runner/work/website/website/_site/blog/index.html":75.1-75.22: 

info warning: Section lacks heading. 

Consider using "h2"-"h6" elements to add identifying headings to all sections.
```

File: _posts/2021-05-24-design-is-your-friend.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title: "Design is Your Friend: Harnessing the Power of Design (and designers) in Open Source"
```
Comment (jdorfman) [inline _posts/2021-05-24-design-is-your-friend.md:3]: @coderberry  add `permalink: /blog/:title/` below `title: ...`

That eliminates the `/2021/05/24/` uri scheme.

File: blog.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+layout: blog
+title: Blog
+slug: blog
+---
+
+<h1 class="h2 text-center mb-4">The Sustain Blog</h1>
+
+<section class="pb-5">
```
Comment (jdorfman) [inline blog.md:9]: Maybe just replace it with a `<div>` - https://www.impressivewebs.com/html5-section/

Comment (jdorfman) [reply blog.md:9]: @coderberry

**Review by jdorfman — APPROVED** [review]

**Comment by jdorfman** [comment]
@coderberry (should be GTG after this) 

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /blog/
Browser:   Chrome 91.0.4472.77 on Mac OS 10.15.7
Viewport:  1536 x 856 @2x
Language:  en-US
Cookies:   Enabled
```

[View on BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-303--sustainoss.netlify.app%2Fblog%2F&os=OS+X&os_version=Catalina&browser=Chrome&browser_version=91.0&resolution=1536x856&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-303--sustainoss.netlify.app/blog/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60b6b7b4aaf95a0007b1cb88&commentID=60b6b9b2d94c2300b28d77de&resolution=resolved)

**Pull request merged**